kvbc/src/Replica.cpp

```diff
@@ -279,12 +282,12 @@ Replica::Replica(ICommunication *comm,
   auto stKeyManipulator = std::shared_ptr<storage::ISTKeyManipulator>{storageFactory->newSTKeyManipulator()};
   m_stateTransfer = bftEngine::bcst::create(stConfig, this, m_metadataDBClient, stKeyManipulator, aggregator_);
   m_metadataStorage = new DBMetadataStorage(m_metadataDBClient.get(), storageFactory->newMetadataKeyManipulator());
+  bftEngine::IControlHandler::instance(new bftEngine::ControlHandler());
   if (!replicaConfig.isReadOnly) {
     stReconfigurationSM_ = std::make_unique<concord::kvbc::StReconfigurationHandler>(*m_stateTransfer, *this);
   }
   // Instantiate IControlHandler.
   // If an application instantiation has already taken a place this will have no effect.
```

Comment:
I think we should move the comment too. Btw, just out of curiosity, what is the reason for moving it?

Reply:
there is no real reason.
I will return it to where it was